src/reducers/users.spec.tsx

```diff
@@ -53,4 +55,36 @@ describe(__filename, () => {
       expect(getCurrentUser(state)).toEqual(null);
     });
   });
+
+  describe('requestLogOut', () => {
+    const prepareRequestLogOut = (params = {}) => {
+      const thunk = requestLogOut({ _logOutFromServer: jest.fn(), ...params });
+
+      const store = configureStore();
+      const dispatch = jest.fn();
+
+      return {
+        dispatch,
+        thunk: () => thunk(dispatch, () => store.getState(), undefined),
+        store,
+      };
+    };
+
+    it('calls logOutFromServer', async () => {
```

Comment:
I was a bit concerned about how we'd test the thunks but these seems to be a pretty good approach.